.circleci/config.yml

```diff
@@ -71,8 +71,8 @@ workflows:
 jobs:
   integration-test-1:
     executor:
-      name: hugo/cibuilds
-      tag: "0.69"
+      name: hugo/default
+      tag: "0.74.3"
```

Comment:
This is a comment not specific to the Hugo orb but rather our orbs in general. I've been having mixed feelings on whether we should use defaults for versions. This leads to us wanting/needing to keep this up to date rather than having a user explicitly choose a version that we can't break.

Reply:
Note this is in the CircleCI config, rather than the orb itself - I'm on the side of "use latest but allow specification of a version", since I don't feel orbs are well suited for the same release cadence as images.

Reply:
Yup - nevermind.